docs/tutorials/pre_executed/ztf-alerts-sne.ipynb

```diff
@@ -0,0 +1,2117 @@
zp4NOnzjDOXP0rPfZpnOe5w6dZ5+vv+B2oa7FteXg766/H2x1a6276H6x4ZLHpca2cW1HLvtePn4l4Mrpq7yrF65FXGu7Hn/95o2UG+03hTc7b+Xcenk7//anO/PvEu4W39O8V3rf8H7lH7Z/7G13az/8IOBBy8PYh3ceCR49fyx//Llj0RPak9KnJk+rO507m7qCui49m/is47n0+afuoj+1/tz4wubFr3/5/dXSM6Gn46XsZd+rZa/1X+964/KmuTeq9/7b3Lef3hW/139f9YH94czHxI9PP03/TPpc9sX2S+PXsK93+3L7+qR8Gb//VwADyqNNOgCvdgFASwKAAc+N1Imq82F/QVRn2n4E/hNWnSH7ixsAtfCfProb/t3cAGDfDgCsoD49BYAoGgBxHgAdO3aoDp7l+s+dykKEZ4OtEV/SctPAvymqM+l3fg9vgVLVBQxv/wVxuILvJuoY/AAAAIplWElmTU0AKgAAAAgABAEaAAUAAAABAAAAPgEbAAUAAAABAAAARgEoAAMAAAABAAIAAIdpAAQAAAABAAAATgAAAAAAAACQAAAAAQAAAJAAAAABAAOShgAHAAAAEgAAAHigAgAEAAAAAQAAAhagAwAEAAAAAQAAAMQAAAAAQVNDSUkAAABTY3JlZW5zaG90odHDIgAAAAlwSFlzAAAWJQAAFiUBSVIk8AAAAdZpVFh0WE1MOmNvbS5hZG9iZS54bXAAAAAAADx4OnhtcG1ldGEgeG1sbnM6eD0iYWRvYmU6bnM6bWV0YS8iIHg6eG1wdGs9IlhNUCBDb3JlIDYuMC4wIj4KICAgPHJkZjpSREYgeG1sbnM6cmRmPSJodHRwOi8vd3d3LnczLm9yZy8xOTk5LzAyLzIyLXJkZi1zeW50YXgtbnMjIj4KICAgICAgPHJkZjpEZXNjcmlwdGlvbiByZGY6YWJvdXQ9IiIKICAgICAgICAgICAgeG1sbnM6ZXhpZj0iaHR0cDovL25zLmFkb2JlLmNvbS9leGlmLzEuMC8iPgogICAgICAgICA8ZXhpZjpQaXhlbFlEaW1lbnNpb24+MTk2PC9leGlmOlBpeGVsWURpbWVuc2lvbj4KICAgICAgICAgPGV4aWY6UGl4ZWxYRGltZW5zaW9uPjUzNDwvZXhpZjpQaXhlbFhEaW1lbnNpb24+CiAgICAgICAgIDxleGlmOlVzZXJDb21tZW50PlNjcmVlbnNob3Q8L2V4aWY6VXNlckNvbW1lbnQ+CiAgICAgIDwvcmRmOkRlc2NyaXB0aW9uPgogICA8L3JkZjpSREY+CjwveDp4bXBtZXRhPgrOJDOOAAAAHGlET1QAAAACAAAAAAAAAGIAAAAoAAAAYgAAAGIAAB1zO9OxSQAAHT9JREFUeAHsnQncDdUbx582tEi0KCrRiooiKtGCImmxVOrTp9KiBSkVEsnSSmkhlKVUCMmS9oWIFG0qkiipaN+1zv/5nf5nmrnvnblz33fue+997+98Pq87c+Y5Z858Z8x55pznPM9mjiZhIgESIAESIAESIIEYCGxGxSIGiqyCBEiABEiABEjAEKBiwQeBBEiABEiABEggNgJULGJDyYpIgARIgARIgASoWPAZIAESIAESIAESiI0AFYvYULIiEiABEiABEiABKhZ8BkiABEiABEiABGIjQMUiNpSsiARIgARIgARIgIoFnwESIAESIAESIIHYCFCxiA0lKyIBEiABEiABEqBiwWeABEiABEiABEggNgJULGJDyYpIgARIgARIgASoWPAZIAESIAESIAESiI0AFYvYULIiEiABEiABEiABKhZ8BkiABEiABEiABGIjQMUiNpSsiARIgARIgARIgIoFnwESIAESIAESIIHYCFCxiA0lKyIBEiABEiABEqBiwWeABEiABEiABEggNgJULGJDyYpIgARIgARIgASoWPAZIAESIAESIAESiI0AFYvYULIiEiABEiABEiABKhZ8BkiABEiABEiABGIjQMUiNpSsiARIgARIgARIgIoFnwESIAESIAESIIHYCFCxiA0lKyIBEiABEiABEqBiwWeABEiABEiABEggNgJULGJDyYpIgARIgARIgASoWPAZIAESIAESIAESiI0AFYvYULIiEiABEiABEiABKhZ8BkiABEiABEiABGIjQMUiNpSsiARIgARIgARIgIoFnwESIAESIAESIIHYCFCxiA0lKyIBEiABEiABEqBiwWeABEiABEiABEggNgJULGJDyYpIgARIgARIgASoWPAZIAESIAESIAESiI0AFYvYULIiEiABEiABEiABKhZ8BkiABEiABEiABGIjQMUiNpSsiARIgARIgARIgIoFnwESIAESIAESIIHYCFCxiA0lKyIBEiABEiABEqBiwWeABEiABEiABEggNgJULGJDyYpIgARIgARIgASoWPAZIIECJPDnn3/KgAED5IADDpBzzjmnTBLYsGGD9OzZUy699FJp0qRJia/x66+/NswGDx4sO+ywQ4nrYwUkUFYJULEoq3eW10UCIQTOPfdcefjhh+XZZ5+V5s2bh0jm76G//vpLjj/+eFm8eLHMmzdPDjvssBJdzO233y69evWSzz//XHbdddcS1cXCJFCWCVCxKMt3l9dGAkkIDB8+XK688krp16+fDBw4MIlE2cn68ssvpX79+uaCoGDstddexb64Aw88UHbeeWd56aWXil0HC5JAIRCgYlEId5nXSAL/J/D8889Lq1at5MgjjzQd5BZbbBHI5qeffpIPP/xQGjRoECiTqQNxnvvFF1+UFi1aSJ06deTNN9+UrbbaKu1mL126VBo2bCj333+/XHjhhWmXZwESKCQCVCwK6W7zWguawD///CP77LOPrFmzRl555RU56qijAnmsWLHCTJFg2P+9994znXKgcMwHMnHu1q1by9NPPy2jRo2SLl26pN3iK664QkaPHi0YAaF9Rdr4WKDACFCxKLAbzsstXAIzZ86
```

Comment:
we dont have any way to monitor the downloading progress, do we?

Reply:
I added a note about dask dashboard